src/System.Windows.Forms/src/System/Windows/Forms/MenuTimer.cs

```diff
@@ -167,7 +165,9 @@ internal void HandleToolStripMouseLeave(ToolStrip toolStrip)
                 // that onmouseleave we make sure there's a selected menu item.
                 if (toolStrip.IsDropDown && toolStrip.ActiveDropDowns.Count > 0)
                 {
-                    ToolStripMenuItem menuItem = (!(toolStrip.ActiveDropDowns[0] is ToolStripDropDown dropDown)) ? null : dropDown.OwnerItem as ToolStripMenuItem;
+                    ToolStripMenuItem? menuItem = (toolStrip.ActiveDropDowns[0] is not ToolStripDropDown dropDown)
+                        ? null
+                        : dropDown.OwnerItem as ToolStripMenuItem;
```

Comment:
Would this be equivalent?
```suggestion
                    ToolStripMenuItem? menuItem = (toolStrip.ActiveDropDowns[0] as ToolStripDropDown)?.OwnerItem as ToolStripMenuItem;
```